diff --git a/electron-builder.yaml b/electron-builder.yaml
index fc32724..2d04554 100644
--- a/electron-builder.yaml
+++ b/electron-builder.yaml
@@ -1,8 +1,8 @@
 directories:
   buildResources: buildResources
   output: dist
-appId: edu.mit.scratch.scratch-desktop
-productName: "Scratch Desktop"
+appId: com.yengawa.mbitmore
+productName: "Microbit More Desktop"
 publish: # empty provider list = don't publish
 mac:
   category: public.app-category.education
@@ -33,9 +33,10 @@ win:
   target:
     - appx
     - nsis
+  publisherName: "Yengawa Lab"
 appx:
-  identityName: ScratchFoundation.ScratchDesktop
-  publisherDisplayName: "Scratch Foundation"
-  publisher: "CN=2EC43DF1-469A-4119-9AB9-568A0A1FF65F"
+  identityName: YengawaLab.MicrobitMoreDesktop
+  publisherDisplayName: "Yengawa Lab"
+  publisher: "CN=YENGAWA-LAB"
 nsis:
   oneClick: false # allow user to choose per-user or per-machine
diff --git a/package.json b/package.json
index 917c56b..cddc5c5 100644
--- a/package.json
+++ b/package.json
@@ -1,8 +1,8 @@
 {
-  "name": "scratch-desktop",
-  "productName": "Scratch Desktop",
-  "description": "Scratch 3.0 as a self-contained desktop application",
-  "author": "Scratch Foundation",
+  "name": "microbit-more-desktop",
+  "productName": "Microbit More Desktop",
+  "description": "Scratch 3.0 desktop application with Microbit-More extension",
+  "author": "Koji Yokokawa",
   "version": "3.15.0",
   "license": "BSD-3-Clause",
   "scripts": {
diff --git a/scripts/afterSign.js b/scripts/afterSign.js
index fbe4fb0..326fe89 100644
--- a/scripts/afterSign.js
+++ b/scripts/afterSign.js
@@ -2,7 +2,7 @@ const {notarize} = require('electron-notarize');
 
 const notarizeMacBuild = async function (context) {
     // keep this in sync with appId in the electron-builder config
-    const appId = 'edu.mit.scratch.scratch-desktop';
+    const appId = 'com.yengawa.mbitmore';
 
     if (!process.env.AC_USERNAME) {
         console.error([
diff --git a/src/main/index.js b/src/main/index.js
index 229d4a4..7b576c5 100644
--- a/src/main/index.js
+++ b/src/main/index.js
@@ -202,7 +202,7 @@ const createAboutWindow = () => {
         height: 400,
         parent: _windows.main,
         search: 'route=about',
-        title: 'About Scratch Desktop'
+        title: 'About Microbit More Desktop'
     });
     return window;
 };
@@ -219,7 +219,7 @@ const createMainWindow = () => {
     const window = createWindow({
         width: defaultSize.width,
         height: defaultSize.height,
-        title: 'Scratch Desktop'
+        title: 'Microbit More Desktop'
     });
     const webContents = window.webContents;
 
